docusaurus/docs/iOS/02-tutorials/02-audio-room.mdx

```diff
@@ -516,7 +516,57 @@ Now when a user requests access to speak the app will look like this:
 
 ### Step 6 - Add microphone control
 
-TODO: add this
+You can read & manage the microphone status from the `call.microphone` published variable. 
```

Comment:
written like this implies that `call.micophone` is a published variable. Better to have `call.microphone.status` or rephrase the sentence.